dotcms-integration/src/test/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImplTest.java

```diff
@@ -4528,4 +4529,56 @@ public void cleanUpExtraTableAfterDeleteContentType() throws DotDataException, D
             ESContentletAPIImpl.setFeatureFlagDbUniqueFieldValidation(oldEnabledDataBaseValidation);
         }
     }
+
+    /**
+     * Method to test: {@link ContentletAPI#checkin(Contentlet, User, boolean)}  }
+     * When: try to save a {@link Contentlet} that don't have the Host set already
+     * Should: populate the Host for the {@link Contentlet} (in this case using the {@link ContentType}'s Host)
+     * and save the siteId in the supporting_values Json field in the unique_fields table
+     *
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @Test
+    public void aaa() throws DotDataException, DotSecurityException {
```

Comment:
Is this the correct name for the test method?

Reply:
done https://github.com/dotCMS/core/pull/30936/commits/fadec93454a793b44e69a61998935b8d78e08a37#diff-bd91db56dfbc6cbcd24831dee1f2e3155c0518a0a2416f9ab5bf8528b83c5b36R4543